environments/daml-sdk-env/build/1_installprerequisites.sh.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Following https://docs.azul.com/zulu/zuludocs/ZuluUserGuide/PrepareZuluPlatform/AttachAPTRepositoryUbuntuOrDebianSys.htm
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 0xB1998361219BD9C9
```

Comment:
I would generally prefer `adoptopenjdk` over Zulu but I’ll leave it up to you. Just seems somewhat more official and it’s clearer how the distributions are built.

Reply:
switched